docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1878,12 +1878,21 @@ is false.
     ```
 
 `<FILE>` is the name of the private key file used to
-[sign Privacy Marker Transactions](../../HowTo/Use-Privacy/Sign-Privacy-Marker-Transactions.md). If
-you do not specify this option, Besu signs each transaction with a different randomly generated
-key.
+[sign Privacy Marker Transactions](../../HowTo/Use-Privacy/Sign-Privacy-Marker-Transactions.md).
 
-If using [account permissioning] and privacy, you must specify a private key file and include the
-corresponding public key in the accounts allowlist.
+!!! note
+
+    This can be the same file used by [`--node-private-key-file`](#node-private-key-file), or a
+    different key file to identify who signed the privacy marker transaction.
+
+You must specify this option if:
```

Comment:
Suggestion: "You must specify this option if you're using:

* a privacy network...
* [account permissioning]..."